src/index.ts

```diff
@@ -64,10 +68,10 @@ const getStoresArray = (stores: Stores): Readable<unknown>[] => {
   return Array.isArray(stores) ? stores : [stores];
 };
 
-export const isLoadable = (object: unknown): boolean =>
+export const isLoadable = <T>(object: unknown): object is Loadable<T> =>
```

Comment:
> object is Loadable<T>

What is this typing feature called? I'm having trouble googling Typescript "is"

Reply:
[Type Predicates](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates)